src/main/java/org/graylog2/gelfclient/GelfConfiguration.java

```diff
@@ -147,6 +151,66 @@ public GelfConfiguration queueSize(final int size) {
     }
 
     /**
+     * Check if TLS option for the transport is enabled.
+     *
+     * @return true if TLS is enabled, false if disabled
+     */
+    public boolean isTls() {
```

Comment:
For clarity I would rename that method to `isTlsEnabled` or `tlsEnabled`. It doesn't follow the Java Beans spec, but this isn't exactly a Java Bean, is it?